hack/jenkins/test-flake-chart/collect_data.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
explain this is only manually
and change it to _manual_

Reply:
Done.